rush.json

```diff
@@ -2,8 +2,8 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
 
-  "npmVersion": "4.5.0",
-  "rushVersion": "5.4.0",
+  "npmVersion": "7.14.0",
```

Comment:
Update the version of npm (primary driver of the usage of components with security issues) and rush